src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationServicesService.java

```diff
@@ -70,6 +79,35 @@ public String sendSystemAdminEmail(List<String> emails, String requesterEmail, A
         }
     }
 
+    public String uploadHtmlFileToAwsS3Bucket(MultipartFile file) {
+        try {
+            MultiValueMap<String, HttpEntity<?>> parts = new LinkedMultiValueMap<>();
+            parts.add("file", new HttpEntity<>(file.getResource(), createFileHeaders(file)));
+
+            return webClient.post()
+                .uri(url + "/notify/upload-html-to-s3")
+                .contentType(MediaType.MULTIPART_FORM_DATA)
+                .body(BodyInserters.fromMultipartData(parts))
+                .attributes(clientRegistrationId("publicationServicesApi"))
+                .retrieve()
+                .bodyToMono(String.class)
+                .block();
+        } catch (WebClientException | IOException ex) {
```

Comment:
Where is the IOException thrown from?

Reply:
I forgot to remove from createFIleHeaders(). thanks